### DISCUSSION TITLE
Post processing

### DISCUSSION THREAD

**Discussion by mdtanvirahmedramim** [godotengine/godot-proposals#4426] [3D]
I want to add post processing effect to some specific Gameobject (Similar to unity3d post processing volume\layer), but each scene can cotain only one World environment node. Any way ?

**Comment by Zireael07** [comment]
Some ways exist, but they boil down to having a second camera in the scene. e.g. https://github.com/jocamar/Godot-Post-Process-Outlines

**Reply by Calinou** [reply]
To be specific, see https://github.com/jocamar/Godot-Post-Process-Outlines/issues/2.
